controllers/ovndbcluster_controller.go

```diff
@@ -581,11 +593,15 @@ func (r *OVNDBClusterReconciler) reconcileServices(
 	}
 
 	// Delete any extra services left after scale down
+	normalServiceLabels := map[string]string{
```

Comment:
normal vs cluster in name. wdyt to use"regular" as type?

Reply:
The problem with "normal" or "regular" is that can be missleading, as it doesn't mean anything inside kubernetes, what's normal or regular? That's why I changed for Cluster (As one of the differences bw headless services and non headless services is that one has Cluster Ip and the other has not). That's why "type: cluster" is the best name I found so far). wdyt?

Also, true, will change normalServiceLabels to clusterServiceLabels, as there's a mismatch on naming :)